pl_examples/basic_examples/dali_image_classifier.py

```diff
@@ -97,11 +100,18 @@ def __init__(
             dynamic_shape=False,
             last_batch_padded=False,
     ):
-        super().__init__(pipelines, size, reader_name, auto_reset, fill_last_batch, dynamic_shape, last_batch_padded)
+        if DALI_VERSION >= '0.28.0':
```

Comment:
lets rather use `from distutils.version import LooseVersion`

Reply:
I've added the change.